framework/logging.cpp

```diff
@@ -2390,6 +2390,7 @@ void YamlLogger::print_result_line()
                 if (loglevel <= sApp->shmem->verbosity && file_log_fd != real_stdout_fd)
                     format_and_print_skip_reason(real_stdout_fd, message);
                 format_and_print_skip_reason(file_log_fd, message);
+                init_skip_message_bytes = init_skip_message.size() + 2;
```

Comment:
The skip message may not be the first message.

Reply:
It does not matter. We do not see if it's first. We only adjust for size here.